backend/test/game/dice_handler_test.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from backend.src.game.dice_handler import DiceHandler
+import backend.src.game.dice.dice as dice
+from mock import Mock
+
+six_mock_dice_values_a = [dice.DieValue.ONE, dice.DieValue.TWO, dice.DieValue.THREE,
+                          dice.DieValue.ATTACK, dice.DieValue.ENERGY, dice.DieValue.HEAL]
+
+six_mock_dice_values_b = [dice.DieValue.ATTACK, dice.DieValue.ENERGY, dice.DieValue.HEAL,
+                          dice.DieValue.ONE, dice.DieValue.TWO, dice.DieValue.THREE]
+
+dice.roll = Mock(return_value=dice.DieValue.ATTACK)
```

Comment:
If you can, set the Mock Attack dice to a more appropriate name, such as MOCK_DIE (constant)
Then in your assertions, you can assert on the value being MOCK_DIE